packages/plugin-css/src/utils/convert-token-name.test.ts

```diff
@@ -0,0 +1,53 @@
+import {describe, expect, test} from 'vitest';
+import {varName} from './convert-token-name.js';
+
+describe('varName', () => {
```

Comment:
Same here—really love this test, but if this were in the `/test/` folder along with the other tests, it’s easier to exclude it from npm (the way the package is currently set up)

Reply:
Resolving this thread to keep this conversation consolidated in [the first thread on this issue](https://github.com/drwpow/cobalt-ui/pull/131#discussion_r1367507831).